<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCreated</class>
 <widget class="QDialog" name="GameCreated">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <widget class="QPushButton" name="closeGameButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>471</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Waiting for player to join...</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourIP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Your IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0.0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readyButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QLabel" name="readyLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Not ready</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
